Documentation/Manuals/User/serializingObjects.md

```diff
@@ -145,15 +145,13 @@ public:
 		BS_RTTI_MEMBER_PLAIN(myString, 2)
 	BS_END_RTTI_MEMBERS
 
-	MyComponentRTTI()
-		:mInitMembers(this)
-	{ }
+	MyComponentRTTI() = default;
 	
 	// ... 
 };
 ~~~~~~~~~~~~~
 
-Field definition portion of the RTTI type always begins with the @ref bs::BS_BEGIN_RTTI_MEMBERS "BS_BEGIN_RTTI_MEMBERS" macro, and ends with the @ref bs::BS_END_RTTI_MEMBERS "BS_END_RTTI_MEMBERS". If field definition is provided you must also provide a constructor that initializes the *mInitMembers* with the *this* pointer.
```

Comment:
Nice catch fixing the docs, I would have forgotten that.

Reply:
I just ran "grep -I -e mInitMembers -r .", so that caught it. I would have forgotten as well.

Though, it wouldn't have caused any harm to initialize in the constructor. Just would have been annoying to users of the API.